src/test/scala/nl/knaw/dans/lib/logging/servlet/MapExtensionsSpec.scala

```diff
@@ -19,7 +19,19 @@ import org.scalatest.{ FlatSpec, Matchers }
 
 class MapExtensionsSpec extends FlatSpec with Matchers {
 
-  "makeString" should "format a Map[String, String] properly" in {
+  "makeString" should "take add square brackets if the input has no elements" in {
```

Comment:
take add: is this proper grammar?

Reply:
reverted the whole 'formatting lists' thing. Not a smart move after all...